fdbserver/workloads/DDMetrics.actor.cpp

```diff
@@ -18,47 +18,67 @@
  * limitations under the License.
  */
 
+#include "fdbclient/ManagementAPI.actor.h"
 #include "fdbclient/NativeAPI.actor.h"
-#include "fdbserver/TesterInterface.actor.h"
-#include "fdbserver/Status.h"
-#include "fdbserver/QuietDatabase.h"
+#include "fdbclient/ReadYourWrites.h"
 #include "fdbserver/ServerDBInfo.h"
+#include "fdbclient/StatusClient.h"
 #include "fdbserver/workloads/workloads.actor.h"
 #include "flow/actorcompiler.h"  // This must be the last #include.
 
 struct DDMetricsWorkload : TestWorkload {
-	double startDelay, ddDone;
+	double ddDone;
+	Value excludeIp;
+	int excludePort;
+	double peakMovingData;
+	double peakInQueue;
+	double peakInFlight;
+	double movingDataPerSec;
 
 	DDMetricsWorkload(WorkloadContext const& wcx)
-		: TestWorkload(wcx), ddDone( 0.0 )
+		: TestWorkload(wcx), ddDone(0.0), peakMovingData(0.0), peakInQueue(0.0), peakInFlight(0.0), movingDataPerSec(0.0)
 	{
-		startDelay = getOption( options, LiteralStringRef("beginPoll"), 10.0 );
+		excludeIp = getOption(options, LiteralStringRef("excludeIp"), Value(LiteralStringRef("127.0.0.1")));
```

Comment:
So simulation doesn't use these IPs. So unless we are just running this locally, I guess this wont do whatever it is supposed to do?

Reply:
This was not intended to be run in simulation. If needed, the proper IP and Port can be supplied using the test text file, since those are just the default values.